routingpy/routers/opentripplanner_v2.py

```diff
@@ -167,30 +167,35 @@ def directions(
         )
         return self._parse_directions_response(response, num_itineraries)
 
+    def _timestamp_to_utc_datetime(self, timestamp):
+        dt = datetime.datetime.fromtimestamp(timestamp / 1000)
+        return dt.astimezone(datetime.timezone.utc)
+
     def _parse_directions_response(self, response, num_itineraries):
```

Comment:
that brings up another good point: this should be static and "public" to be consistent with the other interfaces. we do use this function type in other projects, but also it doesn't need anything from its instance, so it can be static.

Reply:
the same for the isochrones

Reply:
This one would fit nicely in a utils file because it's generic.